extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -125,6 +128,32 @@ class AmpStickyAd extends AMP.BaseElement {
       });
     }
   }
+
+  /**
+   * The function that add a close button to sticky ad
+   * @private
+   */
+  addCloseButton_() {
+    const closeButton = this.getWin().document.createElement('div');
+    closeButton.classList.add('-amp-sticky-ad-close-button');
+    closeButton.setAttribute('aria-label','Close');
```

Comment:
Nit: space after `,`

This requires i18n. We also need to make sure it is perceived as closing the fixed layer. @sriramkrish85 Do you know the right labeling?